app/helpers/form_helper.rb

```diff
@@ -284,7 +284,7 @@ def add_label options, f, attr
     label = options[:label] == :none ? '' : options.delete(:label)
     label ||= ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name) &&
         s_(clazz.gettext_translation_for_attribute_name attr)) if f
-    label = label.present? ? label_tag(attr, "#{label}#{required_mark}".html_safe, :class => label_size + " control-label") : ''
+    label = label.present? ? label_tag(attr, "#{h(label)}#{required_mark}".html_safe, :class => label_size + " control-label") : ''
```

Comment:
Just removing the `.html_safe` should do the same trick w/o needing to workaround getting the ERB helper in the tests.